torch/csrc/distributed/rpc/tensorpipe_agent.h

```diff
@@ -38,12 +38,11 @@ class Context;
 } // namespace transport
 
 namespace channel {
-template <typename TBuffer>
 class Context;
-using CpuContext = Context<CpuBuffer>;
+using CpuContext = Context;
```

Comment:
do we still need `CpuContext` and `CudaContext` has separate types? Are we going to have followup PRs to consolidate these two?

Reply:
No we don't, and in fact these will be removed in a subsequent diff. However this can be done independently in TP/PT so I decided to keep this diff as small as possible.